### PR TITLE
chore: add price graph e2e test

### PR DESCRIPTION
**Pull request by Extheoisah**

### CODE REVIEW

File: e2e/09-price-graph-flow.e2e.spec.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { i18nObject } from "../app/i18n/i18n-util"
+import { loadLocale } from "../app/i18n/i18n-util.sync"
+import { goBack, selector } from "./utils"
+
+describe("Price graph flow", () => {
+  loadLocale("en")
+  const LL = i18nObject("en")
+  const timeout = 30000
+
+  it("should click on price graph button", async () => {
```
Comment (nicolasburtey) [inline e2e/09-price-graph-flow.e2e.spec.ts:10]:
```suggestion
  it("click on price graph button", async () => {
```

the `should` is unnecessary

**Review by nicolasburtey — CHANGES_REQUESTED** [review]
we want to limit the number of file to avoid the file renaming when we created/rename tests

File: e2e/09-price-graph-flow.e2e.spec.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { i18nObject } from "../app/i18n/i18n-util"
```
[the remark this answers is not in the document]
Comment (Extheoisah) [reply e2e/09-price-graph-flow.e2e.spec.ts:1]: Will do that. What would we name the file? `other-tests`?

**Comment by Extheoisah** [comment]
@nicolasburtey I've addressed your comments.

**Pull request merged**